### ISSUE TITLE
Wink Hub 2 consistently unreachable

### ISSUE THREAD

**Issue by joshourisman** [sibartlett/homebridge-wink3#41]
I recently added a Wink Hub 2 to my setup in order to add some z-wave devices. Lately, I've noticed that homebridge is reporting that it's not reachable locally, with the following status:

```
[9/7/2017, 9:27:19 AM] [wink] Checking if hub is reachable (Wink Hub 2, 192.168.86.31)...                 
[9/7/2017, 9:27:24 AM] [wink] Hub is not reachable locally (Wink Hub 2, 192.168.86.31). Error: ESOCKETTIMEDOUT                                                                                                      
[9/7/2017, 9:27:24 AM] [wink] Will continue without local control for hub (Wink Hub 2, 192.168.86.31) 
```

The Wink app doesn't report any issue with it, and the devices paired with it are still controllable with both the Wink app and HomeKit, though oddly I've discovered that I can't use the actual switches to control their devices, I can _only_ control them through one of the apps, which is weird (no idea if that's related).

I haven't started doing any digging to try and figure out what's going on yet, but I figured someone else might have experienced something similar and have a solution or at least direction to go in.

**Comment by joshourisman** [comment]
Possibly relevant, if I GET the IP for the hub, this is the response:

```
HTTP/1.1 401 not authorized
content-length: 33
content-type: application/json
Date: Thu, 07 Sep 2017 14:07:13 GMT
Connection: close

{"description":"not authorized"}
```

**Comment by joshourisman** [comment]
Interestingly, if I GET either of my two Wink Relays, they 500:

```
HTTP/1.1 500 Internal Server Error
X-Content-Type-Options: nosniff
Content-Type: text/html; charset=utf-8
Content-Length: 22
Date: Thu, 07 Sep 2017 14:09:36 GMT
Connection: close

Internal Server Error
```

**Comment by sibartlett** [comment]
What URL are you using to do a GET? It should have port 8888: https://192.168.86.31:8888/

**Comment by joshourisman** [comment]
Yeah, I noticed that after a little digging. If I use 8888 then it does
fail to open a connection to the hub.

**Comment by sibartlett** [comment]
Have you tried restarting the hub? Sometimes a restart gets the web server working.

**Comment by joshourisman** [comment]
You know, having worked in IT for nearly 20 years now, you'd think I'd have tried that. You'd be wrong…

I'll give it a shot after work.

**Comment by sibartlett** [comment]
![](https://media.giphy.com/media/F7yLXA5fJ5sLC/giphy.gif)
